### DISCUSSION TITLE
Comma in path param isn't staying encoded

### DISCUSSION THREAD

**Discussion by mku123** [mitre/HTTP-Proxy-Servlet#203] [Q&A]
Hi 

Our url contains path param which maybe comma separated, it is not staying encoded after proxy, hence we get 404 not found in the response. Is there a workaround we can get this to work? 

A quick response is highly appreciated.

Thanks

**Comment by mku123** [comment]
Found it.

**Reply by dsmiley** [reply]
What did you find?
